queries.sql

```diff
@@ -1,3 +1,92 @@
+-- Commit the transaction.
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+TABLE animals;
+COMMIT;
+TABLE animals;
+-- Delete all records in the animals table, then roll back the transaction.
+BEGIN;
+DELETE FROM animals;
+TABLE animals;
+ROLLBACK;
+TABLE animals;
+-- Delete all animals born after Jan 1st, 2022.
+-- Create a savepoint for the transaction.
+-- Update all animals' weight to be their weight multiplied by -1.
+-- Rollback to the savepoint
+-- Update all animals' weights that are negative to be their weight multiplied by -1.
+-- Commit transaction
+BEGIN;
+DELETE FROM animalsWHERE date_of_birth > '2022-01-01';
+SAVEPOINT sp1;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO sp1;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+TABLE animals;
+COMMIT;
+TABLE animals;
+-- How many animals are there?
+SELECT COUNT(*) FROM animals;
+-- How many animals have never tried to escape?
+SELECT COUNT(escape_attempts) FROM animals WHERE escape_attempts = 0;
+-- What is the average weight of animals?
+SELECT AVG(weight_kg) FROM animals;
+-- Who escapes the most, neutered or not neutered animals?
+SELECT AVG(escape_attempts) FROM animals GROUP BY neutered;
+-- What is the minimum and maximum weight of each type of animal?
+SELECT species,
+  MIN(weight_kg) as MIN_WEIGHT,
+  MAX(weight_kg) as MAX_WEIGHT
+FROM animals
+GROUP BY species;
+-- What is the average number of escape attempts per animal type of those born between 1990 and 2000?
+SELECT species,
+  AVG(escape_attempts) as Average_escape_attempts
+FROM animals
+WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+
+-- day 3
+-- What animals belong to Melody Pond?
+SELECT owners.id, full_name, name FROM owners LEFT JOIN animals ON owners.id = animals.owner_id WHERE full_name = 'Melody Pond';
+
+-- List of all animals that are pokemon (their type is Pokemon).
+SELECT animals.name FROM animals RIGHT JOIN species ON animals.species_id =species.id  WHERE species.name = 'Pokemon'; 
+-- List all owners and their animals, remember to include those that don't own any animal.
+SELECT owners.full_name, animals.name FROM animals RIGHT JOIN owners ON owners.id = animals.owner_id;
+-- How many animals are there per species?
+SELECT species.name, count(species.name) FROM species LEFT JOIN animals ON species.id = animals.species_id GROUP BY species.name;
+-- List all Digimon owned by Jennifer Orwell.
+SELECT animals.name FROM  owners LEFT JOIN animals ON owners.id = animals.owner_id WHERE animals.species_id = 2 AND animals.owner_id = 2; 
+-- List all animals owned by Dean Winchester that haven't tried to escape.
+SELECT owners.full_name, animals.name, animals.escape_attempts FROM owners LEFT JOIN animals ON owners.id = animals.owner_id WHERE animals.escape_attempts = 0 AND owners.full_name = 'Dean Winchester';
```

Comment:
### Project requirements

![image](https://github.com/Sami-Ullah-Tufail/postgresql-vetclinic/assets/78251927/ec582394-5dda-49a3-a1b8-c4edccdc42e7)

As we were expected to add a screenshot of **each of our query results** to the pull request description, I could not find the screenshots of this query. ==> `List all animals owned by Dean Winchester that haven't tried to escape.`

Please add your screenshot. this helps to ensure that you run the queries with success. 🤓🤝

Reply:
i have added the screen shot it is the first one